PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -322,3 +325,13 @@ def miniAOD_customizeAllMC(process):
     miniAOD_customizeCommon(process)
     miniAOD_customizeMC(process)
     return process
+
+def miniAOD_forPhase2(process):
+    print "Switching off electron and photon IDs (not yet working for Phase2 era)"
+    process.patElectrons.addElectronID = False
+    process.patElectrons.electronIDSources = cms.PSet()
+    process.patPhotons.addPhotonID = False 
+    process.patPhotons.photonIDSources = cms.PSet()
```

Comment:
design of eras is to apply modifications in the cfi files where parameters are defined.
It looks like you are planning to violate it from the start.

customie applies when customize method is called.
era modifiers are applied when corresponding file that defines the era is loaded.